layers/+themes/themes-megapack/packages.el

```diff
@@ -37,7 +37,8 @@
     dracula-theme
     espresso-theme
     farmhouse-theme
-    firebelly-theme
+    ;; Removed from melpa
```

Comment:
Maybe add reason for removal, something like "removed from Melpa due to license issues"? Same goes for the rest of the comments.